test/functional/test_framework/messages.py

```diff
@@ -1232,6 +1232,26 @@ def __repr__(self):
         return "msg_mempool()"
 
 
+class msg_notfound:
+    __slots__ = ("vec")
+    command = b"notfound"
+
+    def __init__(self, vec=None):
+        if vec is None:
+            self.vec = []
+        else:
+            self.vec = vec
```

Comment:
nit: more concise (if not exactly equivalent) as `self.vec = vec or []`

Reply:
I think Marco's version is clearer.